docs/processes/quality-assurance/testing-and-dev.md

```diff
@@ -0,0 +1,200 @@
+
+### TBC template link \*\*\*
+
+```block
+will link when relevant PR is merged - doc currently not there
+```
+
+### User Stories
+
+User Stories are user-centric statements often put in the form of:
+
+**"As a [persona], I [want to], [so that]"**
+
+This describes a _persona + need + purpose_.
+
+A **[persona]** – describes the person we are building this for. The persona is not just a role or job title; we want to have an understanding of who the person is, how they work, think and feel.
+
+**[wants to]** – describes the user's _intent_ (what they want to achieve) and not the feature they use. If you are describing implementation specifics here, you are taking the wrong approach.
+
+**[so that]** – explains the motivation behind the user's desire and the overall benefit to them from what they are trying to achieve.
+
+User stories use non-technical language to provide context and an understanding of what is to be built and why. They provide a framework for approaching development which is user-focused, discrete and small enough to be manageably tackled by team members in an Agile environment.
+
+Benefits include:
+
+- Keeps focus on the user: keeps the team focused on solving real-world problems in user-centric ways.
+- Enables collaboration: by defining the end goal, the team can work together on deciding on how best to meet it
+- Drives creative solutions: the focus on the end-user solution required, rather than the how, encourages creative problem-solving (where to-do lists do not).
+
+In Agile methodology, during a sprint or iteration, the team decides which user stories to tackle that story and discuss requirements; once agreed, requirements are added to the stories; Story tasks and subtasks can be tracked and assigned using the team's relevant Trello board; user stories for future sprints can remain in the backlog.
+
+When defining user stories, consider:
+
+- Definition of "done": how will we know when a task/story is complete?
+- Tasks or subtasks: what steps need to be completed and by whom?
+- User personas: who is the user being considered; are there multiple users (consider making user stories for each of them)
+- Ordered steps: In a larger process, write a story for _each step_
+- Time: consider how long a story may take; if it's more than one sprint, or seems very complex, the story may need to be broken down into smaller stories (or, you may be looking at an epic)
+
+Further background on user stories can be found at https://www.atlassian.com/agile/project-management/user-stories
+
+## Testing Templates
+
+[Sample Test Strategy](templates/test-strategy-template.md))
+
+[Test Plan Template](templates/test-plan-template.md)
+
+[Bug Report Template](templates/bug-report-template.md)
```

Comment:
**nitpick:** We can convert this to a bullet lists

```suggestion
- [Sample Test Strategy](templates/test-strategy-template.md))
- [Test Plan Template](templates/test-plan-template.md)
- [Bug Report Template](templates/bug-report-template.md)

```

Reply:
Done - with a new intro sentence, as indented bullets looked a little off as the only thing under a heading.